.github/workflows/neptune-contrib.yml

```diff
@@ -0,0 +1,28 @@
+name: neptune-contrib
+on: [push]
```

Comment:
I think this should on push to master only. 
Is it now? Seems like it is any push.